### DISCUSSION TITLE
Prisma get datetime as a string

### DISCUSSION THREAD

**Discussion by abdulrahimiliasu** [prisma/prisma#14678] [Q&A] [answered]
Suppose I have table in Postgres SQL with prisma.

```prisma
// Prisma Model

model my_table {
    comment   String   @db.VarChar(255)
    date      DateTime @db.Timestamp(6) // Stored as Now() in sql
}
```

After insert into my table I have. (the date is stored with no timezone)
```
// SQL DB Table

| comment  |           date             |
| -------- | -------------------------- |
| Hello    | 2022-08-04 08:16:32.716904 |
```
I want to query here

```typescript
// Prisma Query

const rows = await this.prismaClient.my_table.findMany({
            orderBy: {
               date: 'desc'
            },
        });
```

How can I get back the date as ```'2022-08-04 08:16:32.716904' ``` instead of a `Date` object as the `date.toISOString()` returns `2022-08-04T08:16:32.716Z` and not  `2022-08-04T08:16:32.716904Z` ?

**Comment by abdulrahimiliasu** [comment]
related issue https://github.com/prisma/docs/issues/668

**Comment by abdulrahimiliasu** [comment, marked as answer]
I found a work around, that works

```typescript
const row = await prismaClient.$queryRaw<{date: string}[]>
(Prisma.sql`SELECT CAST(date AS VARCHAR) FROM my_table where name = "Hello"`)
```

This returns the date as `string` without mapping the date to `Date` object.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
